test/events.test.tsx

```diff
@@ -118,11 +118,162 @@ describe('Events', () => {
     await userEvent.pointer({ target: getCellsAtRowIndex(0)[1], keys: '[MouseRight]' });
     expect(getCellsAtRowIndex(0)[1]).toHaveAttribute('aria-selected', 'true');
   });
+
+  it('should call onCellSelected when cell is clicked', async () => {
+    let idx;
+    let rowIdx;
+    let row;
+    let column: CalculatedColumn<Row> | undefined;
+
+    render(
+      <EventTest
+        onCellSelected={(args) => {
+          idx = args.idx;
+          rowIdx = args.rowIdx;
+          row = args.row;
+          column = args.column;
+        }}
+      />
+    );
+    await userEvent.click(getCellsAtRowIndex(0)[0]);
+
+    expect(getCellsAtRowIndex(0)[0]).toHaveAttribute('aria-selected', 'true');
+    expect(idx).toBe(0);
+    expect(rowIdx).toBe(0);
+    expect(row).toStrictEqual({
+      col1: 1,
+      col2: 'a1'
+    });
+    expect(column?.key).toBe('col1');
+  });
```

Comment:
I think we can combine all these tests into one test
```suggestion
    // should call onCellSelected when cell is double clicked
    await userEvent.dblClick(getCellsAtRowIndex(0)[1]);
  });
```

Reply:
You are right. I have combined the tests.